src/apps/gp-v1/pages/Settings.tsx

```diff
@@ -3,7 +3,7 @@ import { useForm, Resolver as FormResolver, FieldErrors, ResolverResult } from '
 import styled from 'styled-components'
 import { DevTool } from 'HookFormDevtool'
 
-import { walletApi } from 'api'
+import { walletApi } from 'api/gpV1'
```

Comment:
and central walletapi also would be kewl

Reply:
Same point as for web3, except, will for instance the explorer need a wallet api?